.kitchen.yml

```diff
@@ -56,6 +56,7 @@ suites:
   - name: backups
     run_list:
       - recipe[test]
+      - recipe[apt]
```

Comment:
If you want to avoid needlessly running this on Centos, You could hoist this to the platform level, adding `run_list` attributes to all the ubuntu platforms.